vipps-getting-started.md

```diff
@@ -45,26 +76,25 @@ Please see the general developer information: https://github.com/vippsas/vipps-d
 
 # Getting a test account (and a production account)
 
+>>>>>>> a89023ee6b2d4a88657165a03423256ecbf7c20f
```

Comment:
Seems like merge conflict have been included in commit.

Reply:
fixed in bab9c39